integration_tests/dbt_project.yml

```diff
@@ -1,5 +1,5 @@
 name: 'qualtrics_source_integration_tests'
-version: '0.1.0'
+version: '0.1.1'
```

Comment:
Why don't we also persist the docs in the integration tests to confirm that this works as expected across all other warehouses as well.

Reply:
@fivetran-joemarkiewicz Good idea! I have added, and it passed integrations tests. One caveat--I had to turn off the [persist_docs: columns for databricks](https://github.com/fivetran/dbt_qualtrics_source/blob/bug/persist-docs/integration_tests/dbt_project.yml#L14) to get it to pass. This is not us since I'm seeing there are [known issues](https://docs.getdbt.com/reference/resource-configs/persist_docs#support) on dbt's end with persist_docs for databricks.